netpyne_ui/netpyne_geppetto.py

```diff
@@ -108,7 +108,32 @@ def importModel(self, modelParameters):
         netpyne_geppetto.simConfig = getattr(simConfigModuleName, modelParameters["simConfigVariable"])
 
         os.chdir(owd)
-
+    
+    def importCellTemplate(self, params):
+        # Get Current dir
+        owd = os.getcwd()
```

Comment:
Most of the lines in this method are redundant with the importModel feature. Can we extract them to a common method?